lib/metrics.js

```diff
@@ -17,9 +17,6 @@ const metricsIds = {
 
 module.exports = {
   hiddenMetrics: [
-    metricsIds.FV,
-    metricsIds.VC100,
-    metricsIds.VC85,
```

Comment:
Yeah but by default I wanna see `['tti', 'psi', 'ttfmp', 'ttfcp']` in chart and all bunch of metrics in file...